School.js

```diff
@@ -12,37 +14,46 @@ class School {
   /**
    * Add class to classes
    * 
-   * @param {string} name - Name of the class
-   * @param {string} teacher - Name of instructor 
-   * @return {Class} Class object
+  //  * @param {string} name - Name of the class
+  //  * @param {string} teacher - Name of instructor 
+  //  * @return {Class} Class object
    */
   addClass(name, teacher) {
     let newClass = new Class(name, teacher);
-    this.classes[name] = newClass;
+    return this.classes[name] = newClass;
   }
 
   /**
    * Enroll student in class
    * 
-   * @param {string} className - Name of the class
-   * @param {Student} student - Student object
-   * @return {Student} Enrolled student
+  //  * @param {string} className - Name of the class
+  //  * @param {Student} student - Student object
+  //  * @return {Student} Enrolled student
    */
-  enrollStudent(className, student) {
-    // Your code here
+  enrollStudent(className, student) { 
+    let classObj = this.classes[className].students
+    let newStudent = new Student(student.name, student.age, student.city, student.grade)
+    return classObj.push(newStudent)
   }
 
+  
+
 
 
 
   /**
    * Get all students enrolled in a class
    * 
-   * @param {string} className - Name of the class
-   * @return {Student[]} Array of Student objects
+  //  * @param {string} className - Name of the class
+  //  * @return {Student[]} Array of Student objects
    */
   getStudentsByClass(className) {
-    // Your code here
+    let classObj = this.classes
+    for(let key in classObj){
+      if(classObj[key].name === className){
```

Comment:
You can just say `if(classObj[className])` that would return true if there is a key with the same class name as the one you are passing as an argument.